shell/src/main/java/org/crsh/lang/impl/groovy/Helper.java

```diff
@@ -33,7 +33,11 @@
 /**
  * @author Julien Viet
  */
-public class Helper {
+public final class Helper {
+	
+  public Helper(){
+	  throw new AssertionError("Must not instantiate this class");
```

Comment:
If this is really necessary, perhaps it should throw an IllegalStateException, instead.